src/boot/axios.js

```diff
@@ -9,6 +10,31 @@ import axios from 'axios';
 // for each client)
 const api = axios.create({ baseURL: '/backend' });
 
+/**
+ * Get default headers for Api, if not provide it use session token store in localstorage.
```

Comment:
Get default headers for Api. If session token is not provided, use session token stored in localStorage.

Reply:
done